CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+### Added
+
+- Add CRs to create bastion host, when creating a CAPI cluster via `template cluster` command.
+- Add configuration to allow SSH for Giant Swarm employees when creating CAPI cluster. Applies to `template cluster` and `template nodepool` commands.
```

Comment:
@marians something like this is enought?

Reply:
Superb, thanks!